src/test/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowServiceTest.java

```diff
@@ -7,21 +7,16 @@
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 import static org.mockito.Mockito.when;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Not an blocking issue, but bringing in the entire util and mockito packages... there is a IntelliJ setting that disables this.

Reply:
Fair, I've updated to single imports